src/it/warResources/verify.groovy

```diff
@@ -20,8 +20,7 @@ globalConfigHelp  = new File(basedir, 'target/war-resources-it/help-globalConfig
 // code uses plexus not maven so escapes are not supported.
```

Comment:
this comment is now incorrect.

I have not enabled escapes - but it is trivial to do so.  this may be more of a breaking change, however I found only a handful of projects (mostly legacy) that used `webResources` and none used filtering so I could make that change here too (it is only an extra line of code)